u8timeseries/backtesting/backtesting.py

```diff
@@ -64,7 +68,7 @@ def backtest_forecasting(series: TimeSeries,
     raise_if_not(start != series.end_time(), 'The provided start timestamp is the last timestamp of the time series',
                  logger)
 
-    last_pred_time = series.time_index()[-fcast_horizon_n - 2] if trim_to_series else series.time_index()[-2]
+    last_pred_time = series.time_index()[-fcast_horizon_n - 1] if trim_to_series else series.time_index()[-1]
```

Comment:
@hrzn This change (from -2 to -1) should be ok, right?

Reply:
Yes, I think that makes sense, I don't remember why I put -2. 🤔

Reply:
Maybe you can add a `raise_if_not` check that `fcast_horizon_n >= 1`

Reply:
done